tests/integration/conftest.py

```diff
@@ -16,7 +16,17 @@
 
 
 @pytest.fixture
-def anonymous_connection(test_endpoints) -> SynapseBaseClient:
+def tiny_file():
+    file_name = "tiny_file.txt"
+    data = os.urandom(200)
+    with open(file_name, 'wb') as writer:
```

Comment:
use temp file

Reply:
I looked into `tempfile` package and couldn't find a function that meets my use case. In this case, I need to return a path of an actual created file, so that the test function could actually read the file with `open`. Available functions in `tempfile` return objects equivalent to object returns from `open`.  It's at a lower level than what I'm looking for for the integration test.